src/os/bluestore/BlueStore.cc

```diff
@@ -5610,6 +5629,12 @@ int BlueStore::_init_alloc(std::map<uint64_t, uint64_t> *zone_adjustments)
       }
     }
 
+    // start with conventional zone "free" (bluefs may adjust this when it starts up)
+    auto reserved = _get_ondisk_reserved();
+    shared_alloc.a->init_add_free(reserved,
+				  bdev->get_conventional_region_size() - reserved);
```

Comment:
Shouldn't this use the same 
p2align(bdev->get_conventional_region_size(), min_alloc_size) - reserved
pattern for the length as mkfs does?